Source/PlasticSourceControl/Private/PlasticSourceControlUtils.cpp

```diff
@@ -659,8 +576,8 @@ static bool RunStatus(const FString& InDir, TArray<FString>&& InFiles, const ESt
 	check(InFiles.Num() > 0);
 
 	TArray<FString> Parameters;
-	Parameters.Add(TEXT("--compact"));
-	Parameters.Add(TEXT("--noheaders"));
+	Parameters.Add(TEXT("--machinereadable"));
+	Parameters.Add(TEXT("--fieldseparator=;"));
```

Comment:
Nit: did you check whether any of the values can contain the character `;`? I would say they can't, but just in case

Reply:
I picked a separator that couldn't be used for file paths nor for file status, move percentage, but do I miss something else?
I'll take the time to review some examples;

header looks like
```
STATUS;41;UEPlasticPlugin;localhost:8087
STATUS;41;UEPlasticPlugin;SRombauts@cloud
```

Reply:
To recap: @SRombautsU and I reviewed this situation and while theoretically there could be branch or repo names that contain `;`, it's not a common scenario. We'll take note of that for the future and continue with that separator, because it's not really related to the task itself and it would take extra time to figure out a clean solution.